internal/db/storage/blob/s3/reader/reader.go

```diff
@@ -53,23 +59,56 @@ func (r *reader) Open(ctx context.Context) (err error) {
 		Key:    aws.String(r.key),
 	}
 
-	r.resp, err = r.service.GetObjectWithContext(ctx, input)
+	r.wg = new(sync.WaitGroup)
```

Comment:
could you please move this line before L82?

Reply:
@kpango thanks. moved.